utilities/infra.py

```diff
@@ -1072,6 +1072,9 @@ def get_node_audit_log_entries(log, node, log_entry):
     ).splitlines()
     has_errors = any(line.startswith("error:") for line in lines)
     if has_errors:
+        if any(line.startswith("404 page not found") for line in lines):
```

Comment:
hints missing for this function?
can we check the return code instead of checking "404 page not found"? checking return code gives reliable information

Reply:
There are couple of things here 
1. I kind of avoid getoutput because it never clearly mentions if error is in stderr or stdout. Error should be in stderr only else i consider it as a bug in negative test cases .
2. getoutput never gives return code in some cases we might want to see it and if !=0 then only start checking if's.
3. there is a possibility that logs might not exist. maybe fetch first, check rc and then search in python

Reply:
@rnetser should we consider replacing grep with python search buildin at later if fetching is successful.

Reply:
The output message is a string  `404 page not found`, not a http response or something like that. This is what openshift responds to the oc command that tries to access to a file that does not exist.

I agree with @rnetser that it's not worth refactoring the entire test. We simply need to skip checking for a file that doesn't exist (because it was rotated since the file list was fetched) and have a suitable event to identify that situation: the '404 page not found'.